include/deal.II/matrix_free/operators.h

```diff
@@ -1091,23 +1115,65 @@ namespace MatrixFreeOperators
           const VectorizedArrayType *               in_array,
           VectorizedArrayType *                     out_array) const
   {
-    const unsigned int given_degree =
-      fe_eval.get_shape_info().data[0].fe_degree;
     if (fe_degree > -1)
       internal::CellwiseInverseMassMatrixImplFlexible<dim,
                                                       VectorizedArrayType>::
         template run<fe_degree>(
           n_actual_components,
-          fe_eval.get_shape_info().data.front().inverse_shape_values_eo,
-          inverse_coefficients,
+          fe_eval,
+          ArrayView<const VectorizedArrayType>(inverse_coefficients.data(),
+                                               inverse_coefficients.size()),
+          false,
           in_array,
           out_array);
     else
       internal::CellwiseInverseMassFactory<dim, VectorizedArrayType>::apply(
         n_actual_components,
-        given_degree,
-        fe_eval.get_shape_info().data.front().inverse_shape_values_eo,
-        inverse_coefficients,
+        fe_eval,
+        ArrayView<const VectorizedArrayType>(inverse_coefficients.data(),
+                                             inverse_coefficients.size()),
```

Comment:
I think you can use this one:
```suggestion
        make_array_view(inverse_coefficients),
```
since we have this function: https://github.com/dealii/dealii/blob/9bd903cea1181de1b3ffa68c1485d1158e60b1ed/include/deal.II/base/array_view.h#L1090-L1093